packages/dd-trace/src/debugger/devtools_client/breakpoints.js

```diff
@@ -47,12 +48,15 @@ async function addBreakpoint (probe) {
     url, lineNumber, columnNumber, probe.id, probe.version
   )
 
+  const condition = probe.when && compileCondition(probe.when)
```

Comment:
what if compileCondition throws? is there a try catch handling of this addBreakpoint?